bib/cmd/bootc-image-builder/util.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
It might make sense to put this in `bib/internal/utils.go` and made the functions public so that they can be used by other internal code in the future.

Reply:
The latest iteration of the code has changed and removed this.